docs/src/whatsnew/3.1.rst

```diff
@@ -257,11 +312,44 @@ This document explains the changes made to Iris for this release
 #. `@pp-mo`_ reworked the netcdf loading code, replacing Pyke rules with a pure Python implementation.
    (:pull: `4198`)
 
-
 #. `@lbdreyer`_ updated the CF standard name table to the latest version: 
    `v77 <http://cfconventions.org/Data/cf-standard-names/77/src/cf-standard-name-table.xml>`_.
    (:pull:`4282`)
 
+=========================================
+Internal changes in v3.0.4 (22 July 2021)
+=========================================
+#. `@bjlittle`_ Unpinned the `cftime`_ package dependency within Iris in order
+   to allow use of the latest versions of `cftime`_, `cf-units`_ and `nc-time-axis`_.
+   (:pull:`4222`)
+
+#. `@rcomer`_ modified test modules so they run consistently under ``pytest`` and
+   ``nose``, and also fixed some minor issues with :class:`~iris.time.PartialDateTime`.
+   (:pull:`4249`)
+
```

Comment:
You have lost the note:
```
   Note that, we are forced to drop support for ``Python 3.6`` in this patch due to
   the third-party package dependencies required by (:pull:`4222`).
```
?

Reply:
There is a note about this in the release highlights, so I thought that might be enough.
I don't think we really need to explain _why_  :  this note in the bugfix release was only there because it had changed requirements from the original v3.0.0